protodroid/src/main/java/id/lukasdylan/grpc/protodroid/Protodroid.kt

```diff
@@ -17,6 +17,7 @@ class Protodroid private constructor(context: Context) {
 
     var loggingEnabled: Boolean = false
     var notifySuccessful: Boolean = true
+    var defaultUniqueErrors: Boolean = false
```

Comment:
can you explain what is this variable does?

Reply:
It defaults the filter to not show duplicates and only show errors, this is useful if you are testing since OK responses and duplicates can become quite noisy